platform/javascript/os_javascript.cpp

```diff
@@ -117,6 +117,17 @@ Error OS_JavaScript::execute(const String &p_path, const List<String> &p_argumen
 	return OK;
 }
 
+Error OS_JavaScript::create_process(const String &p_path, const List<String> &p_arguments, ProcessID *r_child_id) {
```

Comment:
could this be calling `OS_JavaScript::execute`? Or vice-versa, `execute` call `create_process` to be more correct?
They do the same thing in the end in JS (and I don't have short term plans to make it more unix-like).

Reply:
Alternatively, we can change the `create_process()` function to always generate an error message e.g. "Not yet supported." And get the JavaScript library to always return `1`. I could be mistaken, but my assumption is that `create_process()` would be used to call `window.open()`. Not a difficult change to make, but beyond the scope of this PR. In the meantime, we have the compat breaking change in place here and in the JavaScript library.

Reply:
> I could be mistaken, but my assumption is that `create_process()` would be used to call `window.open()`. Not a difficult change to make, but beyond the scope of this PR.

Mmmhhh.. that could actually be a good idea... currently it's implemented by the HTML5 editor, so when you press the run button it can be caught and spawn a new wasm instance to run the game (which is running on the same page though, so no `window.open`, which is instead implemented for `OS::shell_open`).
Since you are also changing the editor "run" to use `create_process` I think it should stay compatible at least in this change.
But I like the idea of using `window.open` in conjunction with process ID and postMessage for debugging in 4.0, so I might implement that for create_process in the future.

Reply:
So, just to be clear, we're going with your original suggestion: point `execute()` to `create_process()`.

Reply:
Do you still want two definitions on the JS side?
```
extern int godot_js_os_execute(const char *p_json);
extern int godot_js_os_create_process(const char *p_json);
```

Reply:
No, I think it's fine to just call `create_process()` from `execute()` and have `create_process()` call `godot_js_os_execute` (basically being the same as `execute` before).

Reply:
Updated as suggested.